resources/public/contracts/src/District0xEmails.sol

```diff
@@ -11,11 +11,11 @@ contract District0xEmails {
 
     mapping(address => string) public emails;
 
-    function setEmail(string _encryptedEmail) {
+    function setEmail(string memory _encryptedEmail) public {
```

Comment:
`external`? I think there are more functions which could be `external` instead of `public`

Reply:
Right